src/components/Tabs/TabsComponent.tsx

```diff
@@ -38,6 +43,7 @@ export default function TabsComponent({ mode, gameData, setGameData }: Props) {
             <Tab>Description</Tab>
             {gameData.parentingGuide ? <Tab>Parenting Guide</Tab> : null}
             {gameData.lesson ? <Tab>Lesson Plan</Tab> : null}
+            <Tab>Game Builds</Tab>
```

Comment:
This tab should only exist if the game has non-WebGL game builds OR the admin is in edit mode